packages/amplify-appsync-simulator/src/index.ts

```diff
@@ -154,6 +155,19 @@ export class AmplifyAppSyncSimulator {
     return this.resolvers.get(`${typeName}:${fieldName}`);
   }
 
+  async clearData(): Promise<object> {
+    const it = this.dataSources.values();
```

Comment:
n.p. on naming, but I initially read this like 'it' in kotlin, which is usually just used in a closure to represent 'the thing I'm operating on' but I realize you probably meant it for 'iterator'. I'd prefer to either call this dataSources, or dataSourceIterator, or something a little more verbose.